lib/network/base.go

```diff
@@ -38,6 +38,7 @@ type NetworkClient interface {
 	Connect(node node.Node) ([]byte, error)
 	GetNodeInfo() ([]byte, error)
 	SendMessage(common.Serializable) ([]byte, error)
+	SendTransaction(common.Serializable) ([]byte, error)
```

Comment:
It would be better if this could take the correct type (a `Transaction`) instead of the generic `Serializable` type.
But given the location of this file, it would probably lead to a circular import, which might hint that this interface should be in a different package. But not an urgent matter.